python/paddle/incubate/autograd/functional.py

```diff
@@ -77,6 +102,18 @@ def vjp(func, xs, v=None):
     return ys, _grad(ys, xs, v)
 
 
+@overload
+def jvp(func: _Func, xs: Tensor, v: TensorOrTensors | None = None) -> Tensor:
+    ...
+
+
+@overload
+def jvp(
+    func: _Func, xs: Sequence[Tensor], v: TensorOrTensors | None = None
+) -> tuple[Tensor]:
```

Comment:
```suggestion
) -> tuple[Tensor, ...]:
```

多个？

以及这里返回值是不是两部分组成的？